tst/test_new.tst

```diff
@@ -51,15 +51,6 @@ gap> OMGetObject(stream);
 (1,2,3)
 gap> CloseStream( stream );
 
-# similar, for file tst/test4.out: gpipe encoding
```

Comment:
I don't understand this. So this was part of `test_new.tst` which was runnable before? In this case, this PR throws away too much. I need to re-check.